src/NServiceBus.Core/Configure.cs

```diff
@@ -126,6 +127,23 @@ internal void Initialize()
             }
         }
 
+        void ConfigureBehaviors()
```

Comment:
> This is a breaking change if the customer replaces behaviors as that replaced behavior is now not by default configured.

only if they reuse the replaced behavior.

Should we release these changes as 5.3 without hotfixing it? It only affects structuremap users when they use `AssertConfigurationIsValid` right?